src/components/Article.jsx

```diff
@@ -1 +1,41 @@
-// export const Article = () => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Article = ({ title, date, text }) => {
+  const today = new Date(date);
+  const dateISO = today.toISOString().slice(0, 10);
+  const dateLocaleString = today.toLocaleDateString('en-US', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  });
+
+  return (
+    <div className="box">
+      <h2 className="title" data-cy="title">
+        {title}
+      </h2>
+
+      <p className="subtitle">
+        <time dateTime={dateISO} data-cy="date">
+          {dateLocaleString}
+        </time>
+      </p>
+
+      <p data-cy="text">
+        {text}
+      </p>
+    </div>
+  );
+};
+
+Article.defaultProps = {
+  date: new Date(),
+  text: 'No text added',
+};
+
+Article.propTypes = {
+  title: PropTypes.string.isRequired,
+  date: PropTypes.instanceOf(Date),
+  text: PropTypes.string,
```

Comment:
aren't they required too?

Reply:
in task no info about what properties are required, i made it optional)